kernel/src/snapshot.rs

```diff
@@ -158,6 +158,8 @@ impl Snapshot {
                 _ => list_log_files(fs_client.as_ref(), &log_url)?,
             };
 
+        println!("Commit files: {:?}", commit_files.iter().map(|f| f.location.clone()).collect::<Vec<_>>());
+        println!("Checkpoint files: {:?}", checkpoint_files.iter().map(|f| f.location.clone()).collect::<Vec<_>>());
```

Comment:
NOTE: Before merging, we should either delete all these `println!`, or change them to `debug!` or similar -- depending on whether we think they're generally useful or just temporary debugging aids.

Reply:
Just wanted to confirm the issue first, working on the fix! Will convert the prints to debug.